test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2039,10 +2039,11 @@ def test_cgroup_periodic_update_check_values(self):
         cput_usage = 0.0
         mem_usage = 0
         vmem_usage = 0
+        time.sleep(8)
         for count in range(10):
+            time.sleep(2)
             # Faster systems might have expected usage after 8 seconds
             # TH3 can take up to a minute
```

Comment:
Probably should move this comment to go with the sleep you moved.

Reply:
Done -- I'll commit once I dealt with any other change you might suggest (and I agree with ;-) ).